src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -52,6 +177,179 @@ def start_rwd(request, format=None):
 @decorators.api_view(['POST'])
 @decorators.permission_classes((AllowAny, ))
 def start_analyze_land(request, format=None):
+    """
+    Starts a job to produce a land-use histogram for a given area.
+
+    Uses the National Land Cover Database (NLCD 2011)
+
+    For more information, see the
+    [technical documentation](https://wikiwatershed.org/
+    documentation/mmw-tech/#overlays-tab-coverage).
+
+    ## Response
+
+    You can use the URL provided in the response's `Location`
+    header to poll for the job's results.
+
+    <summary>
+       **Example of a completed job's `result`**
+    </summary>
+
+    <details>
+
+          {
+            "survey": {
+               "displayName": "Land",
+               "name": "land",
+               "categories": [
+                   {
```

Comment:
Can we clean up the spacing here to be more even?

![image](https://user-images.githubusercontent.com/1430060/30174281-819afc1a-93c8-11e7-9063-a75796ffd88d.png)

Since we use 4-space indentation in our code, we should try and use the same here.

Reply:
I'll normalize these before merging.